pages/trainer.page.php

```diff
@@ -52,15 +52,15 @@
 				</div>
 			</div>
 		  </div>
-		  <button type="submit" class="btn btn-primary">Search</button>
+		  <button type="submit" class="btn btn-primary"><?= $locales->SEARCH ?></button>
 		</form>
 	</div>
 </div>
 </header>
 
 <div class="row area" id="trainersGraph">
 	<div class="col-md-12 text-center">
-		<h2 class="sub-title"><?= $locales->TRAINER_GRAPH ?></h2>
+		<h2 class="sub-title"><?= $locales->TRAINERS_GRAPH ?></h2>
```

Comment:
Please change the key to TRAINER**S**_GRAPH in translation.json for all languages.